index.html

```diff
@@ -1,142 +1,169 @@
+							<li class="css"><a href="https://"><img src="img/css.png" alt=""></a></li>
+							<li class="javascript"><a href="https://"><img src="img/js.png" alt=""></a></li>
+						</ul>
+						<button class="btn">See Project</button>
+					</section>
+				</section>
+				<section class="main_body">
+					<div>
+						<h1 class="header2">About<br>Myself</h1>
+						<p class="body2">I’m a software developer! I can help you build a product , feature or website
+							Look through some of my work and experience! If you like what you see and have a project you
+							need coded, don’t hestiate to contact me.</p>
+						<p class="connect">LET'S CONNECT</p>
+						<ul class="icons">
+							<li class="twitter"><a href="https://"><img src="img/Icon - Twitter.png" alt="Twitter"></a>
+							</li>
+							<li class="linkedin"><a href="https://"><img src="img/Icon - Linkedin.png" alt="Linkedin"></a></li>
+							<li class="medium"><a href="https://"><img src="img/Icon - Medium.png" alt="Medium"></a>
+							</li>
+							<li class="github"><a href="https://"><img src="img/Icon -GitHub.png" alt="GitHub"></a></li>
+							<li class="angelist"><a href="https://"><img src="img/Icon - Angelist.png" alt="Angelist"></a></li>
+						</ul>
+						<button class="btn resume">Get my resume</button>
+					</div>
+				</section>
+				<section class="languages">
+					<ul class="lang_info_head">
+						<li class="lang_info"><span class="lang_arrow">languages<i class="fa-solid fa-angle-down"></span></i>
+							<span class="cycle_js"><a href="http://"><img src="img/Frame 9.png" alt=""></a></span>
+							<span class="cycle_html"><a href="http://"><img src="img/Frame 10.png" alt=""></a></span>
+							<span class="cycle_css"><a href="http://"><img src="img/Frame 11.png" alt=""></a></span>
+						</li>
+						<li class="lang_info"><span class="lang_arrow">Frameworks<i class="fa-solid fa-angle-right"></i></span></li>
+						<span class="sep sep1"><i class="fa-thin fa-horizontal-rule"></i></span>
+						<li class="lang_info"><span class="lang_arrow">Skills<i class="fa-solid fa-angle-right"></i></span></li>
+						<span class="sep"><i class="fa-thin fa-horizontal-rule"></i></span>
+					</ul>
+				</section>
+				<section class="contact">
+					<h2 class="contact-me">Contact me</h2>
+					<p class="info-contact">If you have an application you are interested in developing, a feature that
+						you need built or a project that needs coding. I’d love to help with it </p>
+					<form action="https://formspree.io/f/mvonabpr" method="post" class="form-input">
+						<input class="name" type="text" id="" maxlength="30" placeholder="Enter your name" required>
+						<input class="name" type="email" id="" placeholder="Enter your email" required>
+						<textarea class="name" name="" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+						<input type="submit" value="Get in touch" name="user_name" class="get_in_touch">
+					</form>
```

Comment:
- [ ] At 375px with, your form touches both ends of the screen.
- [ ] Your form overflows the width of the device. These issues can be resolved by assigning relative width in percentages. 
**figma**
![width1](https://user-images.githubusercontent.com/99656687/220562534-e34ec807-b4d6-47e9-8cb0-a2d6c69e8688.png)

**Your design**
![width2](https://user-images.githubusercontent.com/99656687/220562624-c2930792-387a-4ddb-a66d-2e75f24d276d.png)